message/html/compose.js

```diff
@@ -53,8 +55,20 @@ exports.create = function (api) {
       title: when(meta.channel, 'Reply is in same channel as original message')
     })
 
+    var draftPerstTimeout = null
+    var draftLocation = resolve(meta).root
```

Comment:
I think the draft loction should just be `JSON.stringify(location)`  (or the hash of that). That means it will be unique per page. Could merge in meta as well?

This I need to check more closely to remind myself what's in meta and location on a few of the pages

Reply:
would also simplify the logic below I think

Reply:
my thinking was to have keys be something humanly understandable (`/channel#name` would have been better than `:` looking at it now) or already usable by the code (we can already extract titles from message keys).

The only problem I see is having multiple drafts for one channel. that might actually be a requirement but I still find hashes messy here since they don't offer any meaning.